tests/golos.posting_test_api.hpp

```diff
@@ -115,16 +115,17 @@ struct golos_posting_api: base_contract_api {
             ("params", json_str_to_obj(json_params)));
     }
 
-    action_result init_default_params() {
+    action_result init_default_params(name vesting_acc) {
```

Comment:
This method involves setting the default parameters, and passing it to the value of the vesting-contract looks strange. Use the default value in the function.